### PR TITLE
Add FailureMessage to vspheredatacenterconfig status

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml [hunk 1]
```diff
@@ -87,6 +87,11 @@ spec:
                     required:
                     - host
                     type: object
+                  labels:
```
Comment (danbudris) [inline config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml:90]: curious why move this

**Review by mdsgabriel — APPROVED** [review]

File: config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml (hunk 1 above)
Comment (abhay-krishna) [reply config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml:90]: I think this is the autogenerated CRD and it sorted the keys alphabetically.

Comment (jiayiwang7) [reply config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml:90]: it's like auto generated from my side w `make release-manifest`

Comment (danbudris) [reply config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml:90]: ah yeah makes sense

**Pull request merged**